lxd/api_metrics.go

```diff
@@ -332,6 +333,9 @@ func internalMetrics(ctx context.Context, daemonStartTime time.Time, tx *db.Clus
 	// Daemon uptime
 	out.AddSamples(metrics.UptimeSeconds, metrics.Sample{Value: time.Since(daemonStartTime).Seconds()})
 
+	// LXD information
+	out.AddSamples(metrics.Info, metrics.Sample{Labels: map[string]string{"version": version.Version}})
```

Comment:
I quite like it! Also, the [OpenMetrics spec](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#info-1) you pointed to includes an example on how to report for other components versions.

This would be a good fit to export the lxc/qemu versions, btrfs, zfs, ceph version.  #12362 shows a bit what I had in mind before knowing about that `_info` bit.

Thanks for bringing this addition in!

Reply:
> This would be a good fit to export the lxc/qemu versions, btrfs, zfs, ceph version. https://github.com/canonical/lxd/issues/12362 shows a bit what I had in mind before knowing about that _info bit.

Absolutely. For `lxd_lxc_features`, `lxd_kernel_features`, `lxd_driver_versions`, and `lxd_storage_supported_drivers_versions`, I'd add the `_info` suffix. Also, I'll change `lxd_info` to `lxd_server_info`, and include the information you mention in #12362.

Reply:
Excellent, thank you!